schemas/objects/customPortableText.js

```diff
@@ -19,6 +19,7 @@ export default {
                 validation: Rule =>
                   Rule.uri({
                     scheme: ['http', 'https', 'mailto', 'tel'],
+                    allowRelative: true,
```

Comment:
fix for https://github.com/regen-network/regen-registry/issues/801

Reply:
nice